src/test/java/io/jenkins/docker/connector/DockerComputerConnectorTest.java

```diff
@@ -171,7 +171,7 @@ protected void should_connect_agent(DockerTemplate template) throws IOException,
         project.getBuildersList().add(new Shell("whoami"));
         final QueueTaskFuture<FreeStyleBuild> scheduledBuild = project.scheduleBuild2(0);
         try {
-            final FreeStyleBuild build = scheduledBuild.get(60L, TimeUnit.SECONDS);
+            final FreeStyleBuild build = scheduledBuild.get(120L, TimeUnit.SECONDS);
```

Comment:
Note that this code change ought to disappear if this PR branch was rebased on main, as the change is already in main.

Reply:
I'm checking to do the rebase.